scripts/artifacts-building/apt/aptly-vars.yml

```diff
@@ -400,6 +399,66 @@ aptly_mirrors:
     update_flags:
       - "-keyring='trustedkeys.gpg'"
     do_update: "{{ aptly_mirror_do_updates }}"
+- name: elastic-logstash-5.5-ALL
```

Comment:
I'm not sure if it matters, but the indentation here doesn't seem to be consistent with the above entries. `-name` should be indented two more spaces like it is above. i.e.:
```
    - name: <name>
      archive_url: <url>
      ..
```